homeassistant/components/tado/climate.py

```diff
@@ -106,6 +113,12 @@ async def async_setup_entry(
         "set_temp_offset",
     )
 
+    platform.async_register_entity_service(
+        SERVICE_SET_PRESET_MODE,
+        CLIMATE_PRESET_MODE_SCHEMA,
+        "set_preset_mode",
+    )
```

Comment:
Home Assistant climate entities already provide presets, It is not clear why a duplicate functionality is created or needed.

Reply:
I have overlooked this, and will remove the added service.